### ISSUE TITLE
Example todoapp doesn't work when packaged

### ISSUE THREAD

**Issue by odvratnozgodan** [JetBrains/compose-jb#349]
The example **todoapp** doesn't work when packaged or run through IDE with `runDistibutable` task. The cause is a exception related to `java.sql.DriverManager`:
```
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: java/sql/DriverManager
	at com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver.<init>(JdbcSqliteDriver.kt:26)
	at com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver.<init>(JdbcSqliteDriver.kt:15)
	at example.todo.common.database.TodoDatabaseDriverFactoryKt.TodoDatabaseDriver(TodoDatabaseDriverFactory.kt:11)
...
```

Not sure if it's a Compose or SQLDelight issue.

**Comment by arkivanov** [comment]
This should be fixed already, see #381

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6467">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
